modules/rgbd/test/test_normal.cpp

```diff
@@ -129,6 +130,40 @@ float cy = H / 2.f + 0.5f;
 Mat K = (Mat_<double>(3, 3) << focal_length, 0, cx, 0, focal_length, cy, 0, 0, 1);
 Mat Kinv = K.inv();
 
+void points3dToDepth16U(const Mat_<Vec3f>& points3d, Mat& depthMap);
+
+void points3dToDepth16U(const Mat_<Vec3f>& points3d, Mat& depthMap)
```

Comment:
could be enough for this test, I thought to generalize it a bit more when R and T are different by 0 for possible future tests ;)